packages/client/hmi-client/src/page/project/components/tera-project-overview.vue

```diff
@@ -364,12 +364,13 @@ async function processFiles(files: File[], csvDescription: string) {
 			case AcceptedTypes.PDF:
 			case AcceptedTypes.TXT:
 			case AcceptedTypes.MD:
-				return processArtifact(file);
+				return processDocument(file);
 			case AcceptedTypes.PY:
 			case AcceptedTypes.R:
 			case AcceptedTypes.JL:
 				return processCode(file);
 			default:
+				// TODO: Should this be process Artifact?
```

Comment:
@YohannParis @dvince2  please let me know here

Reply:
There is a bit of weirdness here around coding for "where we are now" vs "where the puck is going".  I think in theory right now we can only have PDF `Document`s, and giving a .txt or .md would potentially blow things up.  So right now we may want to have those extensions be `Artifact`.

However eventually I would assume txt, md and even docx would go in as a `Document`.  In this case, `Artifact` becomes the catch all "other" category... but then there are real security/usability concerns around letting people just arbitrarily upload "stuff".

This is a long winded way of saying... I don't know :D.  I lean in favour of option 1 here assuming txt and md blow up documents, but thats not what we'll want in the end?

Reply:
I will throw md and txt to artifacts and leave the default as an a future issue open for discussion